WebServices/TodoWCF/TodoWCF/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Xamarin.Forms" version="3.6.0.264807" targetFramework="portable46-net451+win81" />
```

Comment:
Can you use 3.6.0.220655 instead? All samples in the repo are on that version, which makes it easy to do a find/replace for future mass updates.